server/concepts/Tournament.ts

```diff
@@ -123,6 +123,27 @@ export default class Tournament implements TournamentWithoutMS {
     /* eslint-enable @typescript-eslint/no-non-null-assertion */
   }
 
+  public static async getById(tournamenId: string, token: string) {
+    const tournament = await TournamentRepository.getById(tournamenId);
+    if (!tournament) return null;
+    const enrichedTournament = await Tournament.enrichWithMSUsers(tournament, token);
+    return enrichedTournament;
+  }
+
+  public static async getAll(page :number, pageSize: number, token: string) {
```

Comment:
```suggestion
  public static async getAll(page: number, pageSize: number, token: string) {
```

Detail, but try to use one convection